### ISSUE TITLE
Drop html/semantics/popovers/popover-top-layer-interactions.html

### ISSUE THREAD

**Issue by nt1m** [web-platform-tests/interop#677]
### Test List

https://wpt.fyi/results/html/semantics/popovers/popover-top-layer-interactions.html?label=master&label=experimental&aligned&view=interop&q=label%3Ainterop-2024-popover

### Rationale

This seems to be timing out in Safari just because of a fullscreen subtest, which passes in WebKitTestRunner: https://searchfox.org/wubkat/rev/3dea3f050e72b74b587cf5f4943a60c7b5cb5452/LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-top-layer-interactions-expected.txt#10

This suggests some kind of issue with SafariDriver and fullscreen, which either way are out-of-scope for the popover focus area.

Another option could be splitting out that subtest, though I don't think it's worth the trouble given everything else passes everywhere.

@foolip @jgraham What do you think?

**Comment by nt1m** [comment]
@foolip This subtest tries to open a fullscreen element followed by another one, and it times out trying to open the second one.

FWIW, I'm 100% sure it's unrelated to popover, since all the popover tests successfully run before that (the specific failing subtest is the last one, so none of the other ones are blocked by this one).

**Comment by nt1m** [comment]
This might solve the issue on the WebKit side, but I'm not sure: https://github.com/WebKit/WebKit/commit/f62c0edfc546848f8711146ce8a0a90d1c00ace4

Essentially drops the user gesture requirement for entering fullscreen in WebDriver, in favor of the normal transient activation one.

**Comment by nt1m** [comment]
> This might solve the issue on the WebKit side, but I'm not sure: https://github.com/WebKit/WebKit/commit/f62c0edfc546848f8711146ce8a0a90d1c00ace4

It didn't, I'm still puzzled why this passes in WebKit Test Runner but not SafariDriver...

@foolip If you look at the breakdown of the subtests, it's really just the "stack 2 fullscreen elements" subtest failing: https://wpt.fyi/results/html/semantics/popovers/popover-top-layer-interactions.html?label=experimental&label=master&aligned&view=interop&q=label%3Ainterop-2024-popover

**Comment by zcorpan** [comment]
I guess we can close this now?

**Comment by nairnandu** [comment]
Closing this as per consensus
